kalite/distributed/templates/distributed/base.html

```diff
@@ -38,12 +41,17 @@
 
         <script type="text/javascript" src="{% url 'dynamic_js' %}"></script>
 
-        {# Older versions of IE didn't have a JSON object, so this library adds it back in if needed #}
-        <script type="text/javascript" src="{% static 'js/json2.js' %}"></script>
-        <!-- JS translation file statically included as a back-up, so that if the saved file below isn't available, we don't choke. -->
-        {% if current_language != "en" %}
-        <script type="text/javascript" src="{% static 'js/i18n/en.js' %}"></script>
-        {% endif %}
+        {# Use JS to help IE5.5-IE8 support modern browser features, especially for CSS and PNGs #}
+        <!--[if lt IE 8]>
```

Comment:
We should look at how we can support IE8 then. Is it just the sidebar that is the issue? We should provide some graceful fallback that might be helpful for accessibility too.